Initialtask/src/Command/ProductsImportCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\FileParser;
+use App\Entity\Product;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ProductsImportCommand extends Command
+{
+    protected static $defaultDescription = 'import products from *.csv file';
+
+    private $entityManager;
+    private $container;
+
+    private $processedCount = 0;
+    private $skippedCount = 0;
+    private $successCount = 0;
+
+    public function __construct(EntityManagerInterface $em, ContainerInterface $cont)
+    {
+        parent::__construct();
+
+        $this->entityManager = $em;
+        $this->container = $cont;
+    }
+
+    protected function configure(): void
+    {
+        $this->addOption('testMode', null, InputOption::VALUE_NONE, 'is test mode enabled');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $testMode = $input->getOption('testMode');  // if running:  products:import --testMode  == test mode;
```

Comment:
1. Use separate application service, please.
2. Use single responsibility principle (1 class, 1 responsibility)
3. If the same functionality for import is required to be implemented on UI side, it won't be possible to reuse it